app/controllers/auth/AuthAction.scala

```diff
@@ -146,20 +145,20 @@ class AuthActionImpl @Inject()(
 
       request.session.get(configDecorator.authProviderKey) match {
         case Some(configDecorator.authProviderVerify) => {
-          lazy val idaSignIn = s"${configDecorator.citizenAuthHost}/${configDecorator.ida_web_context}/login"
+          lazy val idaSignIn = s"${configDecorator.citizenAuthHost}/ida/login"
           Redirect(idaSignIn).withSession(
             "loginOrigin"    -> configDecorator.defaultOrigin.origin,
             "login_redirect" -> postSignInRedirectUrl(request)
           )
         }
         case Some(configDecorator.authProviderGG) => {
-          lazy val ggSignIn = s"${configDecorator.companyAuthHost}/${configDecorator.gg_web_context}"
+          lazy val ggSignIn = s"${configDecorator.basGatewayFrontendHost}/bas-gateway/sign-in"
```

Comment:
@vimalsuresh @anujakhare check that gg signing in/out works fine when you have no active session e.g. you navigate to the PTA page directly and not through auth login stub